src/Sulu/Bundle/AdminBundle/Resources/js/components/BlockCollection/BlockCollection.js

```diff
@@ -40,15 +41,20 @@ class BlockCollection extends React.Component<Props> {
 
     fillArrays() {
         const {defaultType, onChange, minOccurs, value} = this.props;
-        const {expandedBlocks} = this;
+        const {expandedBlocks, generatedBlockIds} = this;
 
         if (!value) {
             return;
         }
 
         expandedBlocks.push(...new Array(value.length - expandedBlocks.length).fill(false));
+        generatedBlockIds.push(
+            ...new Array(value.length - generatedBlockIds.length).fill(false).map(() => ++BlockCollection.idCounter)
```

Comment:
is the `.fill(false)` needed here?

Reply:
Yes, `map` seems to skip entries if they are empty:

![Screenshot from 2020-04-17 13-33-34](https://user-images.githubusercontent.com/405874/79565013-139ee000-80b0-11ea-9588-13dddeab9cbc.png)

Reply:
okay this is interesting 🙂